app.js

```diff
@@ -1,6 +1,18 @@
 const express = require('express');
+const mongoose = require('mongoose');
+const bodyParser = require('body-parser');
+require('env2')('./config.env');
+const Property = require('./models/Property');
+
+mongoose.connect(process.env.MONGO_URL);
+
 const app = express();
 
+app.use(bodyParser.urlencoded({ extended: false }));
+app.use(bodyParser.json());
+
+require('./routes/propertyRoutes')(app);
```

Comment:
This is not a good way to set up routes in your app,  [look at this example](https://github.com/superhealthyminds/healthyminds/tree/master/src) for the most efficient and scale-able way to do it

Reply:
Done